ReSwiftTests/StoreTests.swift

```diff
@@ -27,6 +28,25 @@ class StoreSpecs: QuickSpec {
 
         }
 
+        describe("#deinit") {
+
+            it("Deinitializes when no reference is held") {
+                var deInitCount = 0
+
+                autoreleasepool {
+                    let reducer = TestReducer()
+                    let _ = DeInitStore(
+                        reducer: reducer,
+                        state: TestAppState(),
+                        deInitAction: { deInitCount++ }
```

Comment:
Closing Brace Spacing Violation: Closing brace with closing parenthesis should not have any whitespaces in the middle. (closing_brace)